assets/js/googlesitekit/notifications/components/common/CTALink.js

```diff
@@ -45,6 +45,7 @@ export default function CTALink( {
 	dismissExpires = 0,
 	dismissOptions = { skipHidingFromQueue: true },
 	gaTrackingEventArgs,
+	disabled = false,
```

Comment:
This component has the `isSaving` property that controls the button's state. The `dismissed` property also controls the state of the button. Let's probably use the same format for both properties and call this one as `isDismissed` instead of just `dismissed`.

Reply:
Thanks @eugene-manuilov , this has been updated and all instances of `disabled` props changed to `isDisabled`, except on the actual `SpinnerButton` component.

Reply:
I'd suggest/prefer we do the opposite since "is saving" doesn't make sense for a generic CTA link as the action isn't necessarily a save (probably never as it has a link URL). It came from our `SpinnerButton` as @10upsimon said, but even there I think it's incorrect. It should be `inProgress` or something like that. `disabled` is the actual attribute for a disabled element which is why this may be used in some places. Looking at the commit, that doesn't seem to be the case. I see there are quite a few instances of this prop throughout so we're not consistent with the use anyways.